src/node_options.cc

```diff
@@ -684,10 +684,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "the line coverage minimum threshold",
             &EnvironmentOptions::test_coverage_lines,
             kAllowedInEnvvar);
-
-  AddOption("--experimental-test-isolation",
+  AddOption("--test-isolation",
             "configures the type of test isolation used in the test runner",
-            &EnvironmentOptions::test_isolation);
+            &EnvironmentOptions::test_isolation,
+            kAllowedInEnvvar);
+  AddAlias("--experimental-test-isolation", "--test-isolation");
```

Comment:
We should probably add a TODO – unless we're fine keeping the undocumented alias, it's very low maintenance for us after all
```suggestion
  AddAlias("--experimental-test-isolation", "--test-isolation"); // TODO(cjihrig): remove this alias in a semver major.
```